pkg/object/statussynccontroller/statussynccontroller.go

```diff
@@ -214,6 +215,12 @@ func (ssc *StatusSyncController) syncStatusToCluster(statuses map[string]string)
 
 	err := ssc.superSpec.Super().Cluster().PutAndDeleteUnderLease(kvs)
 	if err != nil {
+		resourceExhaustedErrMsg := "trying to send message larger than max "
+		if strings.Contains(err.Error(), resourceExhaustedErrMsg) {
+			logger.Errorf("Size of the Easegress objects is too large to be synchronized. " +
+				"Please reduce objects (output of 'egctl object list') or " +
+				"increase the value of cluster.MaxCallSendMsgSize in server configuration.")
+		}
```

Comment:
It is better not to rely on a specific string for the check, because we cannot get a notification if the string is changed at the Etcd side.

Propose to remove these code and update L224 to something like:

```go
 logger.Errorf("sync status failed, if the message size is too large, please increase the value of cluster.MaxCallSendMsgSize in configuration: %v", err)
```